ta4j-core/src/test/java/org/ta4j/core/analysis/criteria/VersusBuyAndHoldCriterionTest.java

```diff
@@ -98,13 +98,13 @@ public void calculateWithNumberOfBars() {
 
         AnalysisCriterion buyAndHold = new VersusBuyAndHoldCriterion(new NumberOfBarsCriterion());
 
-        assertEquals(6d/6d, buyAndHold.calculate(series, tradingRecord), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(6d/6d, buyAndHold.calculate(series, tradingRecord));
     }
 
     @Test
     public void betterThan() {
         AnalysisCriterion criterion = new VersusBuyAndHoldCriterion(new TotalProfitCriterion());
-        assertTrue(criterion.betterThan(2.0, 1.5));
-        assertFalse(criterion.betterThan(1.5, 2.0));
+        assertTrue(criterion.betterThan(numOf(2.0), numOf(1.5)));
+        assertFalse(criterion.betterThan(numOf(1.5), numOf(2.0)));
     }
 }
```

Comment:
VersusBuyAndHold Test looks good.  After `Num.pow(Num)` is merged we'll want to migrate all of the criteria away from `Math.pow`.

Reply:
Yeah there's 1 indicator left, and a couple of tests cases in the indicators, i'll fix those indicator tests in this PR already. Then we only have 1 indicator left to fix.

Reply:
OOS, #217